awscli/examples/servicediscovery/create-public-dns-namespace.rst

```diff
@@ -0,0 +1,18 @@
+**To create an public DNS namespace**
+
+The following ``create-public-dns-namespace`` example creates an public DNS namespace ``example.com``. ::
+
+    aws servicediscovery create-public-dns-namespace \
+        --name example-public-dns.com \
+        --creator-request-id example-public-request-id \
+        --properties DnsProperties={SOA={TTL=60}}
+
+Output::
+
+    {
+        "OperationId": "gv4g5meo7ndmeh4fqskygvk23d2fijwa-k9302yzd"
+    }
+
+To confirm that the operation succeeded, you can run ``get-operation``. For more information, see `get-operation <https://docs.aws.amazon.com/cli/latest/reference/servicediscovery/get-operation.html>`__ .
```

Comment:
This will need to change depending on if its CLI v2 or v2, and possibly if docs locations are changed again. Are we sure we want to reference this here?